tools/pipeline-runner/run-pipelines.sh

```diff
@@ -13,21 +13,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-DIR="${1:-$PWD}"
+DIRS="$1"
 PIPELINE_REPORT="run-pipelines,0"
 
 if [ -z "$APIGEE_USER" ] && [ -z "$APIGEE_PASS" ]; then
   echo "NO CREDENTIALS - SKIPPING PIPELINES"
-elif [ ! -d "$DIR" ]; then
-  echo "$DIR NOT FOUND"
-  exit 1
-elif test -f "$DIR/pipeline.sh"; then
-  PATH=$PATH:./tools/another-apigee-client ./tools/organization-cleanup/organization-cleanup.sh
-  (cd "$DIR" && ./pipeline.sh;)
-  PIPELINE_REPORT="$PIPELINE_REPORT;$DIR Pipeline,$?"
+elif [ -n "$DIRS" ]; then
+  for DIR in $(echo "$DIRS" | sed "s/,/ /g")
+  do
+    echo "[INFO] DevRel Pipeline: $DIR"
+    if ! test -f  "$DIR/pipeline.sh"; then
+      echo "[ERROR] $DIR/pipeline.sh NOT FOUND"
+      exit 1
+    else
+      PATH=$PATH:./tools/another-apigee-client ./tools/organization-cleanup/organization-cleanup.sh
+      (cd "$DIR" && ./pipeline.sh;)
+      PIPELINE_REPORT="$PIPELINE_REPORT;$DIR Pipeline,$?"
+    fi
+  done
 else
   for TYPE in references labs tools; do
-    for D in "$DIR"/"$TYPE"/*; do
+    for D in "$PWD"/"$TYPE"/*; do
```

Comment:
do you think we should put
```sh
SCRIPT=$(readlink -f "$0")
SCRIPTPATH=$(dirname "$SCRIPT")
```

at the top, then instead of `$PWD` we can use `$SCRIPTPATH/../../$TYPE` so we can run the script from anywhere?

e.g. in a debugging scenario 
```
cd /tmp
sh /home/devrel/tools/pipeline-runner/pipeline.sh
```
in the above case, `$PWD` would be `/tmp`.



(note: this is a trivial problem as we always trigger using `npm run pipelines` or `npm run pipelines --prefix /home/devrel` so PWD is correct, but this change will allow us to future proof if ever we remove NPM since we dont actually use it for much)

Reply:
Great find. Unfortunately the story is slightly more complicated as we copy the run-pipelines script into the docker container and relative paths are therefore no longer accurate. Happy to explore this in a separate go.